src/estimators/bloom_filters.py

```diff
@@ -290,12 +292,13 @@ class FirstMomentEstimator(EstimatorBase):
   METHOD_EXP = "exp"
   METHOD_ANY = "any"
 
-  def __init__(self, method, denoiser=None, weights=None):
+  def __init__(self, method, denoiser=None, noiser=None, weights=None):
```

Comment:
I suppose if you set the default value for noiser=lambda x:x, you could save a few lines of code.

Reply:
Setting the default value of the argument makes my error check unworkable, however I did use the lambda trick in the body of __init__, which did save a few lines of repeated code in other functions. Thanks!